### ISSUE TITLE
More layout options (e.g. 2x2)

### ISSUE THREAD

**Issue by ikester** [norihiro/obs-color-monitor#62]
It would be great to have other layout options, in addition to the vertical stacking. For example, a two dimensional grid.

<img width="1155" alt="Screen Shot 2022-09-26 at 12 49 10 AM" src="https://user-images.githubusercontent.com/150993/192196042-87e26021-2697-4edc-9195-7f08113cde1f.png">

**Comment by playaspec** [comment]
Agreed! At minimum, moving the Zebra to the right of the main image and the False Color to the right of the Vector Scope would allow for larger images for roughly the same screen real estate. As it is now, I can't even read the graticules on a 1440p monitor. Saving those two vertically stacked screens would give me space to cram another widget below.

Ideally, a checkbox in the preferences should enable/disable each screen. Most of the time I just need vector scope and waveform for ongoing monitoring at  glance.

At any rate, _EXCELLENT_ work! Thank you so much for this invaluable tool.
